src/tests/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# Unit tests
+set(UNIT_TESTS
+    annotation_tests
+)
+
+foreach(test ${UNIT_TESTS})
+  add_executable("run_${test}" run_tests.cpp "${test}.cpp")
+  target_include_directories("run_${test}" PRIVATE ${CATCH_INCLUDE_DIR} "${CMAKE_SOURCE_DIR}/src")
+
+  if(CUDA_FOUND)
```

Comment:
Are we compiling this repo for GPUs as well? `COMPILE_GPU` is `off` in root level CMakeLists.txt

Reply:
This is a general tests `CMakeLists.txt` file, while it is used only for `Annotation` for the time being. If I am to replace `marian-server` or `decoder` the sources associated will need GPU as well. It's no-op per bergamot-build configuration and therefore shouldn't get in the way? Can cut off and bring back later when I need as well but I favour keeping.

Reply:
I think it's ok to have an option we're not using that leaves Marian compatibility.